contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,162 @@
+<frontmatter>
+  title: Workflow Automation with Github Actions
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Github Actions
+
+📝 [@alyip98](https://github.com/alyip98)
+
+🔎 
+
+
+## What is Github Actions?
+<!--Explain how X fits into the big picture of SE. Describe it relative to topics the reader is likely to know.-->
+Github Actions is a workflow automation solution that is tightly integrated into the Github ecosystem. It is commonly used to provide end-to-end continuous integration and continuous deployment (CI/CD) directly in your repository. With its tight integration with Github's ecosystem of services, users have the potential to automate more than just CI/CD.
+
```

Comment:
Perhaps you can define the `workflow`, `actions` and `triggers` here instead of at How Does GitHub Actions work? Because I think that these are the basic knowledge that the reader should know before diving into the benefits and so on.

Then the How Does GitHub Actions work section should just be Example GitHub Actions Workflow. 

My 2 cents haha